### ISSUE TITLE
Running mms with jest-mongodb inside a docker container (apple M1)

### ISSUE THREAD

**Issue by rgomide** [nodkz/mongodb-memory-server#583]
## Versions

- NodeJS: 10.15.3 (node:10.15.3 image)
- mongodb-memory-server-*: 7.3.6
- mongodb: mongo:4.2.15 (image)
- mongoose: 5.3.0
- system: MacOS 12.0.1 (M1)

package: mongo-memory-server

## What is your question?

I'm trying to run jest tests with jest-mongodb (2.1.0) in a container using docker-compose. When I ran with the standard settings, I got this 404 error:

```
    Status Code is 403 (MongoDB's 404)
    This means that the requested version-platform combination doesn't exist
      Used Url: "https://fastdl.mongodb.org/linux/mongodb-linux-arm64-debian92-latest.tgz"

```

Adding this environment variable (`MONGOMS_DOWNLOAD_URL=https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-4.4.6.tgz`) ir runs with this error:

```
    Instance Exited before being ready and without throwing an error!

      at MongoInstance.once (../node_modules/mongodb-memory-server-core/src/util/MongoInstance.ts:273:13)
      at MongoInstance.closeHandler (../node_modules/mongodb-memory-server-core/src/util/MongoInstance.ts:454:10)
```

Have I missed some step?

**Comment by hasezoey** [comment]
from what i know, docker's system can only be linux, and also the `node:10.15.3` image is based off debian 9.2 (debian92), and so MMS tries to download a debian(linux) image with arch arm64, which does not exist

and then with `MONGOMS_DOWNLOAD_URL=https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-4.4.6.tgz` it downloads & tries to start a macos build, which cannot be run inside linux (which docker is basically)

TL;DR: from what i know, docker cannot run macos applications and a arm64 build for debian does not exist

Workaround: either do no use docker, or use a docker container that is based off ubuntu, because a mongodb build for arm64 exist for ubuntu

**Comment by rgomide** [comment]
I created a new docker file with ubuntu:18.04 image and worked!

```
FROM ubuntu:18.04

ARG DEBIAN_FRONTEND=noninteractive

RUN apt-get update
RUN apt-get -y install curl gnupg
RUN curl -sL https://deb.nodesource.com/setup_10.x  | bash -
RUN apt-get -y install nodejs
RUN npm install

# Install webapp dependencies
WORKDIR /usr/src/app
COPY ../../package*.json ./
RUN npm install

# Run webapp server
EXPOSE 3037
CMD ["npm","run","start-local"]
```
